4-projects/modules/infra_pipelines/cloudbuild_builder/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Google LLC
```

Comment:
It might be worthwhile thinking about sharing the image/s created in bootstrap, so it is consistent throughout the app pipelines

Reply:
ack, I have opened https://github.com/terraform-google-modules/terraform-google-bootstrap/issues/91 to track